setup.py

```diff
@@ -13,7 +13,7 @@
   download_url = 'https://github.com/bodyshopbidsdotcom/basegateway/tarball/{0}'.format(CURRENT_VERSION),
   keywords = ['api', 'gateway', 'http', 'REST'],
   install_requires = [
-    'requests==2.9.1'
+    'requests>=2.20.1'
```

Comment:
actually why requiring that it should be `2.20.1` or newer? it's already been tested with `2.9.1` and `2.14.2`, so we can safely make it `>=2.9.1`

Reply:
because it is not compatible with awsebcli, docker, docker-compose

Reply:
ohh right